xenbus/xb.mli

```diff
@@ -145,3 +146,8 @@ val is_selectable : t -> bool
 val get_fd : t -> Unix.file_descr
 
 val debug : t -> string
+
+(* WARNING: Functions below this point are to be used in testing only *)
+
```

Comment:
There is a convention to hide such functions after 
```
(**/**)
```

in the MLI file. See `strings.mli` in OCaml as an example.